src/Silverback.Integration/Messaging/Messages/MessageLogger.cs

```diff
@@ -13,77 +14,82 @@ public class MessageLogger
     {
         #region Generic
 
-        public void LogTrace(ILogger logger, string logMessage, IRawBrokerEnvelope envelope) =>
-            Log(logger, LogLevel.Trace, null, logMessage, new[] { envelope });
+        public void LogTrace(ILogger logger, EventId eventId, string logMessage, IRawBrokerEnvelope envelope) =>
```

Comment:
Is there a reason why this should remain a class and to be extension methods of ILogger?

Reply:
No, I think it doesn’t have any dependency anymore and it should finally be transformed into a static class... this would also save a dependency in quite a few types.

Reply:
I refactored it.